src/content/blog/2024_qcoder.md

```diff
@@ -0,0 +1,160 @@
+---
+title: QCoder - A platform for quantum competitive programming
+author: Kein Yukiyoshi
+day: 20
+month: 10
+year: 2024
+tags: 
+  - quantum algorithms
+  - qiskit
+  - python
```

Comment:
```suggestion
tags: 
  - guest post
  - microgrant
  - qiskit
  - python
```

Reply:
I have commited the suggestion. Thank you!